app/src/main/java/com/stevesoltys/seedvault/transport/backup/PackageService.kt

```diff
@@ -134,6 +137,14 @@ internal class PackageService(
         }
     }
 
+    private fun PackageManager.getInstalledPackagesCompat(flags: Int): MutableList<PackageInfo> {
+        return if (SDK_INT >= TIRAMISU) {
```

Comment:
Usually, we use the integer here, because who can remember all those desert names?

Reply:
Btw., If there's no more issue with Roboelectric, I also don't mind bumping minSdk to 33.

Reply:
Hmpf, I am getting `android.content.pm.PackageParser$PackageParserException: seedvault/app/build/intermediates/apk_for_local_test/debugUnitTest/apk-for-local-test.ap_ (at Binary XML file line #7): Requires newer sdk version #33 (current version is #29)
` with minSdk 33.

and if settings tests to use 33, I get `API level 33 is not available`. So maybe just leave the deprecated method until repoelectric can handle 33?

Reply:
Killed as discussed